docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -26,361 +26,245 @@ Recommended configuration: 384kB Ram + 8MB SPIRAM + 4MB Flash
+It needs  a working installation of docker follow [Docker Installation](https://docs.docker.com/engine/install/) to install Docker. If running linux make sure to go throught the post installation [steps](https://docs.docker.com/engine/install/linux-postinstall/)
 
-- [create a new robot in the Viam app](#create-a-new-robot)
-- [generate a new project from Viam's micro-RDK template](#generate-a-new-project-from-the-micro-rdk-template)
-- [upload the project to your ESP32](#upload-the-project-and-connect-to-your-esp32-board)
-
-### Create a New Robot
-
-Navigate to [the Viam app](https://app.viam.com) and create a new robot in your desired location.
-Keep your `Mode` and `Architecture` selections at default, and skip the instructions in the **Setup** tab for now as the setup instructions there are not for microcontrollers.
-
-### Generate a New Project from the Micro-RDK Template
-
-Use [the micro-RDK template](https://github.com/viamrobotics/micro-rdk-template.git) to create a new micro-RDK project to upload to your ESP32 by running:
+#### Homebrew
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-cargo generate --git https://github.com/viamrobotics/micro-rdk-template.git
+brew install viamrobotics/brews/canon
 ```
 
-If you would like, you can use `mkdir` to initialize a new repository in the directory you created by running `cargo-generate`, to track any changes you make to the generated project.
+#### Direct Install
 
-You will be prompted to paste your Viam robot configuration information (`viam.json`) into the terminal.
+Go 1.19 or newer must be installed.
 
-To obtain this, navigate to [the Viam app](https://app.viam.com).
-Click the **Copy viam-server config** button on the right side of the **Setup** tab of your robot.
-The Micro-RDK uses the config for communication with the Viam app.
-Paste this into your terminal.
-
-{{< alert title="Caution" color="caution" >}}
-
-All of the generated files should be safe to commit as a project on Github, with the exception of `viam.json`, since it contains a secret key.
-
-{{% /alert %}}
-
-### Modify the Generated Template
-
-You can find the declaration of the robot in the generated file `src/main.rs`.
-This example exposes one GPIO pin (pin 18), and one analog reader attached to GPIO pin 34.
-Change the example as needed.
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go install github.com/viamrobotics/canon@latest
+```
 
-You can, for example:
+Make sure your GOBIN is in your PATH. If not, you can add it with something like: `export PATH="$PATH:~/go/bin"` Note: This path may vary. See [Go](https://go.dev/ref/mod#go-install) for details.
 
-{{%expand "Expose other pins" %}}
+### Setting up development environment manually
 
-Once you have selected an appropriate GPIO pin, according to the pinout diagram included with your ESP32, you can add to the collection of exposed pins.
+#### Install dependencies

```

Comment:
@npmenard is this the only section that is intended to be part of the new "Setting up development environment manually" info? I'm a bit confused about the transition from that to installing the ESP-IDF steps, so some more clarification here could be good

Reply:
I change the name to `Install build decencies` to make it a bit clearer. We need this tools to build the project without the Docker image (the docker has all these tools already installed)